doc/cephfs/fs-volumes.rst

```diff
@@ -1340,5 +1340,28 @@ set with this id was present in the database
 
   $ ceph fs quiesce fs1 sub1 sub2 sub3 --set-id="external-id" --if-version=0
 
+
+.. _disabling-volumes-plugin:
+
+Disabling Volumes Plugin
+------------------------
+By default, on a Ceph cluster, volumes plugin is enabled as well as set as
+``always on`` module. However, in certain cases it might be necessary/favourable
+to disable it. For example, when a CephFS is in a degraded state, the volumes
+plugin commands may get accumulated in MGR instead of getting served. Which
```

Comment:
may accumulate in the Managers instead
and the Manager becomes
In this event, the ``volumes`` plugin can be disabled even though it is ``always on``.

Reply:
Done.